docs/graphql/manual/actions/types/index.rst

```diff
@@ -141,6 +141,11 @@ a scalar called ``Date``, you can define it like.
 These scalars can be used as arguments of the mutation or as fields of object
 types and input types.
 
+.. admonition:: Postgres scalars
+
+   Postgres base types are reusable as custom scalars without explicitly declaring them.
```

Comment:
@rakeshkky do you think we need to explain a little more what this means? Like that the PG base types will be exposed as GraphQL scalars by Hasura? Since I don't understand this in depth, it's hard for me to make specific suggestions. Maybe @lexi-lambda can help?

Reply:
@marionschleifer I tried to improve the documentation in the latest commit.

Reply:
Oh, didn't see that, sorry!